lib/wallaby/element.ex

```diff
@@ -76,12 +76,20 @@ defmodule Wallaby.Element do
   """
   @spec click(t) :: t
 
-  def click(%__MODULE__{driver: driver} = element) do
+  def click(%__MODULE__{driver: driver} = element, retry_count\\0) do
     case driver.click(element) do
       {:ok, _} ->
         element
       {:error, :stale_reference} ->
         raise Wallaby.StaleReferenceException
+      {:error, :obscured} ->
+        if retry_count > 4 do
+          raise Wallaby.ExpectationNotMet, """
+          The element you tried to click is obscured by another element.
+          """
+        else
+          click(element, retry_count+1)
+        end
```

Comment:
(although I now see more and more the need/possibility to have a retry hex.pm package (I also have soem version of retry implemented in benchee))